doc/version_history.rst

```diff
@@ -6,6 +6,17 @@
 Version History
 ===============
 
+v0.23.5
```

Comment:
I think this probably deserves a minor release! Can you make this v0.24.0, please?